CHANGELOG.md

```diff
@@ -31,12 +31,34 @@
 ; #object[reitit.core$single_static_path_router]
 ```
 
+* `:options` requests are served for all routes by default with 200 OK to better support things like [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing)
+  * the default handler is undocumented (for Swagger)
```

Comment:
I would maybe rephrase this as `is not documented in Swagger`. I get what you're saying, but it sounds like the feature is undocumented in the reitit docs.